apps/minifront/src/components/shared/balance-selector.tsx

```diff
@@ -1,19 +1,19 @@
 import { MagnifyingGlassIcon } from '@radix-ui/react-icons';
-import { useState } from 'react';
+import { useId, useState } from 'react';
```

Comment:
All the changes in this file are related to making the modal appear to grow out of the button that's clicked. There are no functional/logical changes here.